src/styles/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 10px;
+  width: $square;
+
+  @each $weekday, $mumber in $weekset {
+    &--start-day--#{$weekday} :first-child {
+      margin-left: #{($width + ($border * 2) + $gap) * $number};
+    }
+  }
```

Comment:
looks like it doesn`t work properly margin is always 206
<img width="1434" alt="image" src="https://user-images.githubusercontent.com/66749919/202843747-34b26520-bd11-42f3-bb7c-a222aea8406c.png">